src/basic/selinux-util.c

```diff
@@ -66,32 +69,34 @@ bool mac_selinux_use(void) {
 #endif
 }
 
+bool mac_selinux_enforcing(void) {
+#if HAVE_SELINUX
+        return selinux_status_page_failed ? (security_getenforce() != 0) : (selinux_status_getenforce() != 0);
```

Comment:
switched to a function pointer.
I initialized it to `security_getenforce()` instead of using NULL, cause this function should only be called if SELinux is enabled.
I was considering adding
```
# should only be called if SELinux is enabled
assert(cached_use > 0)
```
to `mac_selinux_enforcing()`